docs/pages/docs/features/remote-caching.mdx

```diff
@@ -55,6 +55,33 @@ If you are building and hosting your apps on Vercel, then Remote Caching will be
 
 Please refer to the [Vercel documentation](https://vercel.com/docs/concepts/git/monorepos#turborepo?utm_source=turborepo.org&utm_medium=referral&utm_campaign=docs-link) for instructions.
 
+### Artifact Integrity and Authenticity Verification
+
+You can enable Turborepo to sign artifacts with a secret key before uploading them to the Remote Cache. Turborepo uses `HMAC-SHA256` signatures on artifacts using a secret key you provide.
+Turborepo will verify the remote cache artifacts' integrity and authenticity when they're downloaded.
+Any artifacts that fail to verify will be ignored and treated as a cache miss by Turborepo.
+
+To enable this feature, set the `remoteCache` options on your turbo config to include the `signature` option:
+
+```jsonc
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "remoteCache": {
```

Comment:
I think we might need a section defining the `remoteCache` similar to  https://turborepo.org/docs/features/pipelines#defining-a-pipeline

Reply:
depends on how we want to introduce `--remoteOnly` and `teamId` enforcement from `turbo.json`